web-components/src/components/alert-banner/AlertBanner.ts

```diff
@@ -56,7 +56,7 @@ export namespace AlertBanner {
             <md-button
               class="md-alert-banner__close"
               hasRemoveStyle
-              ariaLabel="Close"
+              ariaLabel="Close Banner"
```

Comment:
why hard coded ariaLabel? it should be passed from application. so that it can be localized